src/arch/xtensa/xtos/int-medpri-dispatcher.S

```diff
@@ -73,12 +74,16 @@ LABEL(_Level,FromVector):
 	movi	a2, PS_WOECALL4_ABI|PS_UM|PS_INTLEVEL(XCHAL_EXCM_LEVEL)
 	s32i	a4, a1, UEXC_a4
 	s32i	a5, a1, UEXC_a5
-/* Save also a15 (becomes a11 after _entry) to store coprocessors. */
-	s32i	a15, a1, UEXC_a15
 	wsr.ps	a2
 	rsync
 
-#ifdef __XTENSA_CALL0_ABI__
+	/* store ps and pc */
```

Comment:
We need to comment what we are changing in xtensa HAL and why. Does this also align with latest CAVS version ??

Reply:
Every single operation in the assembly code is commented.
What do you mean by "latest CAVS version"?